app/helpers/application_helper.rb

```diff
@@ -121,3 +121,10 @@ def insert_extras(body)
   end
 
 end
+
+module render_map(lat, lon, items)
```

Comment:
Actually this could go above the `end` on line 123 -- and just start with `def render_map(lat, lon, items)` instead of being a module. And then it'd have to render a partial template in a file at `/app/views/map/_map_helper.html.erb` -- containing these lines: https://github.com/publiclab/plots2/blob/master/app/views/sidebar/_related.html.erb#L80-L108

This is a little complex, because that actually contains two types of maps -- ones based on `@node.lat` type objects, and one on `@maps.first.lat` type objects. But if you replace the code in `_related.html.erb` with a call to your new helper function, you just pass those particular types in, so the top becomes:

```ruby
render_map(@node.lat, @node.lon, @maps)
```

and the bottom becomes:

```ruby
render_map(@maps.first.lat, @maps.first.lon, @maps)
```

Does that make sense? It'll take some more reworking to get right, but I'm happy to review.